daemon/cmd/fqdn.go

```diff
@@ -566,6 +570,7 @@ func (d *Daemon) notifyOnDNSMsg(lookupTime time.Time, ep *endpoint.Endpoint, epI
 		// Add new identities to the ipcache after the wait for the policy updates above
 		d.ipcache.UpsertGeneratedIdentities(newlyAllocatedIdentities)
 
+		stat.DataplaneTime.End(true)
 		endMetric()
```

Comment:
Shouldn't `stat.DataplaneTime.End(true)` be inside `endMetric`?

Reply:
I decided not to since `endMetric` gets called from atleast one other place (see line 457). I wasn't sure what happens if you call end on a span that was never started.

I'm ok either way.